ee/clickhouse/sql/test/__snapshots__/test_schema.ambr

```diff
@@ -150,6 +150,9 @@
       created_at DateTime64(6, 'UTC')
       
       
+  , _timestamp DateTime
```

Comment:
unrelated to the scope but we're still defaulting to `_offset` without `_partition`. I guess it's somewhat helpful but not entirely